### DISCUSSION TITLE
QR CODE

### DISCUSSION THREAD

**Discussion by stunspotx** [Cyfrin/foundry-full-course-cu#1026] [Q&A]
How do I scan the QR code for each lesson to answer the questions to get the NFTs?

**Comment by 0xusmanf** [comment]
[Here](https://github.com/Cyfrin/foundry-full-course-f23/discussions/135) is the list of all the links to challenges. Instead of scanning just click on the link and it will direct you to the challenge.

**Reply by stunspotx** [reply]
Thanks a lot for the reply. Highly appreciated!
